### PR TITLE
[CENNSO-192] Adjust tool for our use-case

### PR DESCRIPTION
**Pull request by sj14**
[CENNSO-192]

[CENNSO-192]: https://travelping.atlassian.net/browse/CENNSO-192?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Related PR: https://github.com/travelping/cennso/pull/376

### CODE REVIEW

**Review by aerfio — COMMENTED** [review]
Please run `go mod tidy` to clean up go.{mod,sum} files

Also maybe adjust readme a bit, stating that it's a fork of XYZ repo? IDK how should we treat LICENSE in this case etc @hudymi

File: pkg/builder/builder.go [hunk 1]
```diff
@@ -136,15 +191,33 @@ func (b *ModelBuilder) Output() error {
...
+		name = name[idx+1:]
+	}
+	name = strings.TrimSuffix(name, "[]")
+	return name
+}
+
+// Return the number of markdown headings (#) by counting the dots from the full name.
+func headings(name string) string {
+	count := strings.Count(name, ".")
+	return strings.Repeat("#", count)
```
[the remark this answers is not in the document]
Comment (sj14) [reply pkg/builder/builder.go:208]: There is always at least one fixed `#` in the template.

[the remark this answers is not in the document]
Comment (sj14) [reply pkg/builder/builder.go:208]: `kind` gets passed to this function:

https://github.com/travelping/crdoc/blob/406b7dcfd36fe0557aa2d9e23406f35de883e7a1/pkg/builder/builder.go#L52

https://github.com/travelping/crdoc/blob/406b7dcfd36fe0557aa2d9e23406f35de883e7a1/pkg/builder/builder.go#L93

[the remark this answers is not in the document]
Comment (aerfio) [reply pkg/builder/builder.go:208]: OK

**Comment by sj14** [comment]
> Also maybe adjust readme a bit, stating that it's a fork of XYZ repo?

When this is needed, we can always do this in a separate PR as it will probably lead to more review feedback, e.g. from technical writers, further postponing the merge of the functionality itself.

**Review by aerfio — APPROVED** [review]
Ok, seperate PR is fine by me, let's unblock this PR and some other PRs

**Pull request merged**